example-notebooks/binary-classifier/usage_config.yml

```diff
@@ -37,5 +37,7 @@ data_usage:
       window_hr: 6
       offset_hr: 0
       usage: target
+      # How to combine multiple *scores* for a context_id when analyzing this event
+      combine_strategy: max
```

Comment:
Should we have a comment in the example that max is the default value?

Reply:
Here I was trying to be explicit on what it does since it may be unintuitive; it would get pretty verbose if we add full descriptions for everything.  
The doc string in the config/model specifies the default which also gets revealed in sphinx docs